scapy/layers/inet.py

```diff
@@ -447,7 +447,8 @@ def hashret(self):
         if self.dst == "224.0.0.251":  # mDNS
             return struct.pack("B", self.proto) + self.payload.hashret()
         if conf.checkIPsrc and conf.checkIPaddr:
-            return (strxor(inet_aton(self.src), inet_aton(self.dst))
```

Comment:
Are the changes to `inet.py` necessary?

Reply:
At a time `inet_aton` was failing on my computer. As it is deprecated, I updated the one with `inet_ntop`. Not related to this PR directly, do you want me to remove it ?